src/components/manage-your-account/index.njk

```diff
@@ -72,7 +67,20 @@
         ]
     }) }}
 
-    <h3 class="govuk-heading-m">{{ 'pages.manageYourAccount.deleteAccount.heading' | translate }}</h3>
+    <h2 class="govuk-heading-m">{{ 'pages.manageYourAccount.manageEmailSubscriptions.heading' | translate }}</h2>
+    <p class="govuk-body">{{ 'pages.manageYourAccount.manageEmailSubscriptions.info' | translate }}</p>
+    <a href="https://www.gov.uk/email/manage" class="govuk-link govuk-body">{{ 'pages.manageYourAccount.manageEmailSubscriptions.link' | translate }}</a>
```

Comment:
should we put in an environment specific link for this, so integration points to integration for example?